syncer-overview.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Syncer 使用文档
-aliases: ['/docs-cn/stable/syncer-overview/','/docs-cn/v4.0/syncer-overview/','/docs-cn/stable/reference/tools/syncer/']
+aliases: ['/docs-cn/stable/syncer-overview/','/docs-cn/v4.0/syncer-overview/','/docs-cn/stable/reference/tools/syncer/','/zh/tidb/stable/syncer-overview','/zh/tidb/dev/syncer-overview']
```

Comment:
```suggestion
aliases: ['/docs-cn/stable/syncer-overview/','/docs-cn/v4.0/syncer-overview/','/docs-cn/stable/reference/tools/syncer/','/zh/tidb/stable/syncer-overview']
```

Reply:
![image](https://user-images.githubusercontent.com/41534398/114125438-0e721b00-9929-11eb-89ff-6749d7b9e121.png)
The master branch does not have this alias.